src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/InProcess/EditorInProcess_Classification.cs

```diff
@@ -128,7 +121,7 @@ public async Task VerifyGetClassificationsAsync(IEnumerable<ClassificationSpan>
             }
         }
 
-        Assert.Equal(expectedArray.Length, actualArray.Length - actualOffset);
+        Assert.Equal(expectedArray.Length, actualArray.Length);
```

Comment:
This assert is not doing anything any more: If the lengths weren't equal, we would have got index out of range exceptions on line 104/5.

I think this should be moved in front of the loop, just for good measure, and to provide a little more information than an index out of range exception would, should it ever fail.